packages/tab/TabRipple.tsx

```diff
@@ -20,47 +20,41 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-import React from 'react';
-import PropTypes from 'prop-types';
+import * as React from 'react';
 import classnames from 'classnames';
-import {withRipple} from '@material/react-ripple';
+import * as Ripple from '@material/react-ripple';
+
+export interface TabRippleProps extends React.HTMLProps<HTMLDivElement>, Ripple.InjectedProps<HTMLDivElement> {
```

Comment:
I'm confused, because I don't see `InjectedProps` in `master`, but I don't see a PR for `typescript-ripple`.